style.css

```diff
@@ -60,10 +68,53 @@ li {
   color: #6070ff;
 }
 
-.navbar ul {
+.desktop-nav {
   display: none;
 }
 
+.mobile-menu {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  top: 0;
+  visibility: hidden;
+  background: #6070ff;
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
#
- Provide the requested design in your background menu.

Reply:
Thanks, @carlos-ssh pls can you be more specific about the background menu design, I don't seem to understand what you requested me to do.